marco_site/templates/marco_site/includes/navbar.html

```diff
@@ -16,6 +16,8 @@
 <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
   <ul class="nav navbar-nav navbar-right">
     <li class="{% active 'planner' %}"><a href="{% url 'planner' %}"><span class="icon-marco oversized"></span>&nbsp; MARINE PLANNER</a></li>
+    {% menu 1 %}
+    {% menu 2 %}
     <li class="dropdown {% active '^/(ocean-stories|data-catalog|data-gaps)/' %}">
```

Comment:
Do the old, hard-coded menus need to come out, or are you leaving them for comparison for now?